### PR TITLE
Update README to use python3 only

### PR DESCRIPTION
**Pull request by ftyers**
Related to issue #1298.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ See the output of `deepspeech -h` for more information on the use of `deepspeech
 
 ## Prerequisites
 
-* [Python 2.7](https://www.python.org/)
+* [Python 3.5](https://www.python.org/)
```
Comment (seanfcarroll) [inline README.md]: I would change this to Python 3.6, or just leave it as Python.

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Project DeepSpeech is an open source Speech-To-Text engine, using a model traine
 
 ![Usage](images/usage.gif)
 
-Pre-built binaries that can be used for performing inference with a trained model can be installed with `pip`. Proper setup using virtual environment is recommended and you can find that documented [below](#using-the-python-package).
+Pre-built binaries that can be used for performing inference with a trained model can be installed with `pip3`. Proper setup using virtual environment is recommended and you can find that documented [below](#using-the-python-package).
```
Comment (seanfcarroll) [inline README.md]: "pip3" is OK, but you could also just say "pip", assuming users are on Python 3. Just a suggestion, either are fine.

Comment (ftyers) [reply README.md]: I'm a bit worried about using just `pip` because sometimes python2 can raise it's head. :)

File: README.md (hunk 1 above)
Comment (ftyers) [reply README.md]: Ok, good idea, I'll do that.

**Comment by lissyx** [comment]
Taken care of in #1336

**Pull request closed without merging**